qemu/tests/qemu_guest_agent.py

```diff
@@ -3933,13 +3933,25 @@ def get_qga_pkg_path(self, qemu_ga_pkg, test, session, params, vm):
         if self.gagent_src_type == "url":
             gagent_host_path = params["gagent_host_path"]
             gagent_download_url = params["gagent_download_url"]
+            mqgaw_ver = match = re.search(r'(?:\d+\.){2}\d+', gagent_download_url)
+            mqgaw_ver_list = list(map(int, mqgaw_ver.group(0).split('.')))
+            if mqgaw_ver_list >= [105, 0, 1]:
```

Comment:
```suggestion
            src_qgarpm_path = params["src_qgarpm_path"]
            cmd_get_qgamsi_path = params["get_qgamsi_path"]
            if mqgaw_ver_list >= [105, 0, 1]:
                src_qgarpm_path = params["src_qgarpm_path_new"]
                cmd_get_qgamsi_path = params["get_qgamsi_path_new"]
```

Reply:
Here are two different values assigned
            if mqgaw_ver_list >= [105, 0, 1]:
                src_qgarpm_path = params["**src_qgarpm_path_new**"]
                cmd_get_qgamsi_path = params["**get_qgamsi_path_new**"]
            else:
                src_qgarpm_path = params["**src_qgarpm_path**"]
                cmd_get_qgamsi_path = params["**get_qgamsi_path**"]

Reply:
Yes, we have 2 different value for a parameter.

If mqgaw_ver_list >= [105, 0, 1], the value should be ***_new, otherwise the value should be ***.
I just thinking set a default value for the parameter and only changing it when the If condition is met.

It's okay for me to keep current code logic, thanks @6-dehan

Reply:
uh, I got your point. let me make it.